tests/conftest.py

```diff
@@ -311,8 +313,6 @@ def app_config(database):
         "docs.backend": "warehouse.packaging.services.LocalDocsStorage",
         "sponsorlogos.backend": "warehouse.admin.services.LocalSponsorLogoStorage",
         "billing.backend": "warehouse.subscriptions.services.MockStripeBillingService",
-        "billing.api_base": "http://stripe:12111",
-        "billing.api_version": "2020-08-27",
```

Comment:
This was a nice change that allowed us to ignore `dev/environment` for tests, which is restored by using the same compose service `base` vs the new and now removed `tests`. Not a blocker.